cli/cmd/testdata/install_controlplane_tracing_output.golden

```diff
@@ -735,7 +736,7 @@ spec:
     spec:
       nodeSelector:
         kubernetes.io/os: linux
-      
```

Comment:
What caused this unrelated whitespace change, I wonder?  Weird........

Reply:
ugh, i think i might have a VS Code plugin that strips whitespace from blank lines, sorry. i can try to back that out.